### DISCUSSION TITLE
Debian backport support for USB hotplugging?

### DISCUSSION THREAD

**Discussion by DaanSelen** [FreeRDP/FreeRDP#10995] [General]
I was testing around in our Debian installation and it all works very well but I noticed with:
```
xfreerdp3 "$rdpFile" /u:"${username}" /p:"${password}" /drive:hotplug,*
```
That my device does not show up when taking it out and reinserting it. It does successfully mount again on the Linux system, but does not get resend to the RDP-session.
```
user@Cubic:~$ xfreerdp3 --version
This is FreeRDP version 3.5.1 (n/a)
```
Does anyone have suggestions?

**Comment by akallabeth** [comment]
well, you need to request a backport or bugfix at the debian bug tracker.

don´t remember when exactly that was fixed, but 3.7.0 changelog mentions something that could match.
